### ISSUE TITLE
Module not found: Error: Can't resolve 'fsevents'

### ISSUE THREAD

**Issue by Arcath** [kentcdodds/mdx-bundler#4]
- `mdx-bundler` version: 1.1.3
- `node` version: 14.15.4
- `npm` version: 6.14.10
- `next` version: 10.0.5

Relevant code or config

[prepare-mdx.ts](https://github.com/Arcath/arcath.net-next/blob/main/lib/functions/prepare-mdx.ts) and [mdx.ts](https://github.com/Arcath/arcath.net-next/blob/main/lib/components/mdx.tsx)

What you did:

Tried to build my Next.JS site on my Windows machine.

What happened:

Running `next build` results in this output:

```
info  - Creating an optimized production build  
Failed to compile.

ModuleNotFoundError: Module not found: Error: Can't resolve 'fsevents' in 'C:\Users\adam\Documents\GitHub\arcath.net-next\node_modules\mdx-bundler\node_modules\rollup\dist\shared'


> Build error occurred
Error: > Build failed because of webpack errors
```

Reproduction repository:

Sorry its not very minimal but my full site is here: https://github.com/Arcath/arcath.net-next
The files using `mdx-bundler` are [prepare-mdx.ts](https://github.com/Arcath/arcath.net-next/blob/main/lib/functions/prepare-mdx.ts) and [mdx.ts](https://github.com/Arcath/arcath.net-next/blob/main/lib/components/mdx.tsx)

Problem description:

Build doesn't work on Windows but does on Vercel. There is a seperate issue I need to diagnose which is what prompted me to do a local build.

Suggested solution:

Possibly a change to rollup config? This feels like it could be a rollup issue when running undex next.js.

**Comment by kentcdodds** [comment]
I'm thinking about switching to https://github.com/wooorm/xdm and using esbuild rather than rollup to bundle. Should be *much* faster and xdm has fewer bugs and requires no client-side runtime 👍

Just gotta find the time to do that.

**Comment by Arcath** [comment]
This has been fixed by #5.

Confirmed against version 2.0.0.
